museum/sampleRecords/locationHistory.destination.json

```diff
@@ -0,0 +1,245 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "movement_id":"1",
+      "id":"1",
+      "idno":"1",
```

Comment:
More of a follow-up question, but do these really even need `idno`?  Or is it just mandatory in CA?  I realise it's a legacy ID but maybe that should be stored in a `legacyId` field instead?  I'm sure it's probably fine just wondering why you put this in `idno` really.